src/packages/@ncigdc/tableModels/utils.js

```diff
@@ -0,0 +1,101 @@
+// @flow
+import React from "react";
+import _ from "lodash";
+import Tooltip from "@ncigdc/uikit/Tooltip/Tooltip";
+import { tableToolTipHint } from "@ncigdc/theme/mixins";
+import { DATA_CATEGORIES } from "@ncigdc/utils/constants";
+import styled from "@ncigdc/theme/styled";
+import { Th, Td } from "@ncigdc/uikit/Table";
+import * as Links from "@ncigdc/components/Links/RepositoryLink";
+import { makeFilter } from "@ncigdc/utils/filters";
+import { findDataCategory } from "@ncigdc/utils/data";
+
+const NumTh = styled(Th, { textAlign: "right" });
+const NumTd = styled(Td, { textAlign: "right" });
+
+export const createDataCategorySubColumns = (
+  type,
+  typeCount = type + "_count",
+  Link = Links[`Repository${_.capitalize(type)}sLink`]
+) => {
+  let idFilter; // because of weird ternary bug https://github.com/babel/babel/issues/5809
+
+  if (type === "case") {
+    idFilter = node => ({
+      field: "cases.project.project_id",
+      value: node.project_id
+    });
+  } else {
+    idFilter = node => ({
```

Comment:
Can we pass this in from the call location instead of internalizing it into the function?